src/snapred/backend/dao/request/SpecifyNormalizationRequest.py

```diff
@@ -0,0 +1,11 @@
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+
+class SpecifyNormalizationRequest(BaseModel):
+    runNumber: str
```

Comment:
this seems to be the same as NormalizationCalibrationRequest except the members are in a different order (one with a slightly different name)